### PR TITLE
ENTMQIC-2560 undeliverable-here

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

**Comment by pwright** [comment]
@grs  can't see to add you as reviewer, but could you take a look?

File: docs/books/modules/user-guide/how-routers-route-messages.adoc [hunk 1]
```diff
@@ -64,6 +64,12 @@ Flow control is "real" in that credits flow across the link route from the recei
 * Server-side selectors
 +
 With a link route, consumers can provide server-side selectors for broker subscriptions.
+
+* Modified delivery states
```
Comment (grs) [inline docs/books/modules/user-guide/how-routers-route-messages.adoc:68]: I would use something like 'consumer specific negative acknowledgement' as the 'title' here. Modified states in general are not an issue it is the consumer scoped undeliverable-here flag that is the issue.

**Review by grs — COMMENTED** [review]
One suggested change, otherwise looks ok to me.

**Comment by k-wall** [comment]
Looks reasonable to me, thanks @pwright

**Comment by gemmellr** [comment]
Seems like it should either have a DISPATCH Jira reference, or not have a ref.

**Comment by pwright** [comment]
closed in favor of https://github.com/apache/qpid-dispatch/pull/867

**Comment by gemmellr** [comment]
No need to close PRs for such things, you can just rename it and/or force push if updating commits.

**Pull request closed without merging**